website/docs/plugins/ray_launcher.md

```diff
@@ -0,0 +1,233 @@
+---
+id: ray_launcher
+title: Ray Launcher plugin
+sidebar_label: Ray Launcher plugin
+---
+<!-- Add PyPI links etc -->
+
+The Ray Launcher plugin provides 2 launchers: `ray_aws` and `ray_local`. `ray_aws` launches jobs remotely on AWS and is built on top of [Ray Autoscaler](https://docs.ray.io/en/latest/autoscaling.html). `ray_local` will launch jobs on your local machine. 
+
+
+This plugin requires Hydra 1.0 to install:
+```commandline
+$ pip install hydra-ray-launcher --pre
+```
+
+After installation, override the Hydra launcher via your command line to activate one of the launchers:
+
+```commandline
+# AWS Ray Launcher
+$ python my_app.py hydra/launcher=ray_aws
+
+# Local Ray Launcher
+$ python my_app.py hydra/launcher=ray_local
+```
+
+#### ray_aws launcher
+
+`ray_aws` launcher is built on top of ray's [autoscaler cli](https://docs.ray.io/en/latest/autoscaling.html). To get started, you need to 
+config your AWS credentials first, tutorials can be found [here](https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-files.html).
+You can run an initial check on credential configuration running the following command.
+```commandline
+ python -c 'import boto3;boto3.client("ec2")'
+```
+:::caution
+`ray autoscaler` expects your AWS user has certain permissions for `EC2` and `IAM`. This [issue](https://github.com/ray-project/ray/issues/9327) provides some contexts.
+:::
+
+
+`ray autoscaler` expects a yaml file to provide specs for the new cluster, which we've schematized in `hydra_ray_launcher.hydra_plugins.hydra_ray_launcher.conf.__init__.RayClusterConf`, 
+The plugin defaults are in `conf/hydra/launcher/ray_aws.yaml`. You can override the default values in your app config or from command line.
+
+:::caution
+This plugins depends on `cloudpickle`, as a result please make sure your local machine and the AWS cluster runs the same version of: `ray`, `hydra-core` and `python`. You can 
+easily install the desired software version by overriding `RayClusterConf.setup_commands`.
+:::
+
+Now we can go ahead and run `train.py` using `ray_aws` launcher
+
+```commandline
+$ tree -L 1
+.
+├── conf
+├── model
+└── train.py
+
+$ python train.py --multirun hydra/launcher=ray_aws +ray_mode=aws random_seed=1,2,3 
```

Comment:
seems odd.
Why do we need ray_mode at all if the launcher is already aws specific?
can it just be folded into the ray_aws config?

Reply:
yea. mostly because RayAWS mode have extra   [example app specific configs](https://github.com/facebookresearch/hydra/pull/518/files#diff-405bbcb7589e354a58188c0541b40bc7e29430a4d4bc28fab6fbb516da000feeR15-R25) that doesn't actually fit with the default settings.


maybe I could split the examples into 2 apps, one that doesn't have the need to download and upload files and the other that needs that.

I want to address this as a follow up, thoughts?